src/client/app_configuration_http.rs

```diff
@@ -131,14 +158,25 @@ mod tests {
         example_configuration_enterprise: Configuration,
         configuration_feature1_enabled: Configuration,
     ) {
-        let mut client = {
+        let (mut client, metering_recv) = {
             let live_cfg_mock = LiveConfigurationMock {
                 configuration: example_configuration_enterprise,
             };
 
-            AppConfigurationClientHttp {
-                live_configuration: live_cfg_mock,
-            }
+            let configuration_id = ConfigurationId::new(
+                "test_guid".to_string(),
+                "test_env_id".to_string(),
+                "test_collection_id".to_string(),
+            );
+            let (metering, metering_recv) = start_metering_mock(configuration_id);
```

Comment:
We need to provide some `ConfigurationId` here and it is not the same one we used for the `Configuration` objects... there will be a mismatch (see `FIXME` below). In following PRs, if we think it's needed, we can check it is consistent somewhere inside, this test will fail, and it will be great to remove those `FIXME`s. I don't think it's something we should worry about in this PR.

Reply:
I think this is not an issue for now.